docs/how-to/visual-studio-setup.md

```
@@ -0,0 +1,45 @@
+
+如果你的开发环境也受此影响，可以尝试将项目目录添加到杀毒软件的排除项中。以下以 Windows 平台常见的 Microsoft Defender 为例说明操作步骤：
+
+1. 打开 **Windows 安全中心**。
+2. 进入 **病毒和威胁防护** > **“病毒和威胁防护”设置** > **管理设置**。
+3. 向下滚动到 **排除项**，点击 **添加或删除排除项**。
+4. 点击 **添加排除项**，选择 **文件夹**，然后将本仓库的根目录（`lf2-taiwu-mods`）完整添加进去。
+5. **重启计算机** 使设置完全生效。
+
+> [!WARNING]
+> 将文件夹添加到排除项会使其免受病毒扫描，这意味着其中潜藏的恶意软件将不会被检测到。请确保你信任该文件夹中的所有文件来源，并自行承担相应风险。
+
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)